aiidalab_widgets_base/structures.py

```diff
@@ -872,3 +877,49 @@ def remove(self, _):
         atoms = self.structure.copy()
         del [atoms[self.selection]]
         self.structure = atoms
+
+
+class OptimadeQueryWidget(ipw.VBox):
```

Comment:
I would prefer to have the widget definition in a distinct module.

Reply:
But this way is also fine since this is how most widgets are defined right now.

Reply:
maybe move to the `databases` module as @CasperWA mentioned earlier.

Reply:
As opposed in the general `databases` file, where only `CodQueryWidget` is now?